apis_core/history/urls.py

```diff
@@ -0,0 +1,34 @@
+from apis_core.history import views
+from apis_core.history.api_views import (
+    EntityHistoryLogs,
+    GenericHistoryLog,
+    TempTripleHistoryLogs,
+)
+from django.urls import include, path
+from rest_framework import routers
+
+app_name = "history"
+
+
+router = routers.DefaultRouter()
+router.register(r"entity/edit_log", EntityHistoryLogs, "entityhistorylog")
+router.register(r"temp_triple/edit_log", TempTripleHistoryLogs, "temptriplehistorylog")
+
+urlpatterns = [
+    path(
+        "change_history/<contenttype:contenttype>/<int:pk>/",
+        views.ChangeHistoryView.as_view(),
+        name="change_history",
+    ),
+    path(
+        "add_new_version/<contenttype:model>/<int:pk>/",
```

Comment:
s/model/contenttype (or `content_type`?)

Reply:
used `contenttype` for consistency